saleor/discount/utils/voucher.py

```diff
@@ -304,22 +325,45 @@ def get_products_voucher_discount(
     return total_amount
 
 
+def create_or_update_voucher_discount_objects_for_order(
```

Comment:
I think that this method should go to `saleor/discount/utils/order.py`. It might also help with circular imports

Reply:
Yes, but you can have methods just voucher specific, and that it what I understand should be in `voucher` file. The methods that applied directly on `Order` I would see in `Order` file